src/context/SøknadContext.tsx

```diff
@@ -200,6 +200,17 @@ const [SøknadProvider, useSøknad] = createUseContext(() => {
     }));
   };
 
+  const fjernBarnFraSøknad = (id: string) => {
+    const nyBarneListe = søknad.person.barn.filter((b: IBarn) => b.id !== id);
+
+    settSøknad((prevSoknad: ISøknad) => {
+      return {
+        ...prevSoknad,
+        person: { ...søknad.person, barn: nyBarneListe },
+      };
+    });
```

Comment:
Har nok ikke så mye å si, men for å være helt trygg på at vi ikke oppdaterer med en gammel state kan vi flytte denne inn
```suggestion
    settSøknad((prevSoknad: ISøknad) => {
      const nyBarneListe = prevSoknad.person.barn.filter((b: IBarn) => b.id !== id);
      return {
        ...prevSoknad,
        person: { ...søknad.person, barn: nyBarneListe },
      };
    });
```